ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js

```diff
@@ -1,63 +1,64 @@
 jest.unmock('../../../core/service/oauth-helper.js')
 jest.unmock('../../../core/service/percent-encoder.js')
-// var OauthHelper = require('../../../core/service/oauth-helper.js')
+jest.unmock('crypto-js')
+jest.unmock('crypto-js/hmac-sha1')
```

Comment:
These two lines are the only change of the file, except for the un-commented code